services/streammanager/lambda/lambdamanager.go

```diff
@@ -15,14 +16,13 @@ import (
 
 // Config is the config that is required to send data to Lambda
 type destinationConfig struct {
-	InvocationType string
-	ClientContext  string
-	Lambda         string
+	InvocationType string `json:"invocationType"`
+	ClientContext  string `json:"clientContext"`
+	Lambda         string `json:"lambda"`
 }
 
 type inputConfig struct {
```

Comment:
The name `inputConfig`, is it still relevant?

Reply:
yes, it is to hold what transformer is sending right?

Reply:
It's the input payload, right? Now, we are not even sending the config.

Reply:
https://github.com/rudderlabs/rudder-server/blob/master/services/streammanager/common/common.go#L16
we are expecting json as input in the method.
https://github.com/rudderlabs/rudder-transformer/blob/master/src/v0/destinations/lambda/transform.js#L66
We are sending this structure from transformer.

Reply:
I'm just talking about the name `inputConfig`. Should it be something like `inputData` at least?

Reply:
Sorry I missed reading the name part. inputData is fine

Reply:
Updated